nextcord/user.py

```diff
@@ -143,8 +153,15 @@ def default_avatar(self) -> Asset:
         """:class:`Asset`: Returns the default avatar for a given user.
 
         This is calculated by the user's discriminator.
+
+        ..versionchanged:: 2.6
+            Indexes avatar using ID instead of discriminator.
```

Comment:
Not too sure about the wording here. Indexing is never mentioned to the end user. Maybe something like this?

```suggestion
            Added handling for the new username system for users without a discriminator.
```